docs/current_docs/index.mdx

```diff
@@ -62,7 +62,7 @@ Dagger SDKs provide resources for developing Dagger modules using a familiar lan
 
 ### Modules
 
-Dagger Modules are platform-agnostic software components that consume or extend Dagger type
+Dagger modules are platform-agnostic software components that consume or extend Dagger types.
```

Comment:
arguable that this usage is almost like GitHub Actions, the name of the feature/system versus a particular software component a GitHub action.

Reply:
But looking at it in situ, it feels right like this
![image](https://github.com/user-attachments/assets/f4bf6ccb-8921-443f-8ed6-cfde00431392)